Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ubuntu:16.04
+MAINTAINER Devin Ekins <devops@keen.io>
+
+# Setup the dependencies
+RUN apt-get update -y
+RUN apt-get install -y apt-utils
+RUN apt-get install -y libtool make automake
+
+# Copy local repo into the container
+ADD . /twemproxy
+WORKDIR /twemproxy
+
+# Install Twemproxy
+RUN autoreconf -fvi && \
+    ./configure --enable-debug=full && \
+    make
+
+# Expose Twemproxy Ports
+EXPOSE 22122 22222
```

Comment:
Turns out those docs are stating you can use multiple ports in one line, optionally. Not: you can use array notation to express ports to expose.  I've tried to get the latter to work several times and have looked online, and failed to discover anyone using it.  I'm pretty sure it isn't used for this.